main.go

```diff
@@ -71,6 +73,7 @@ func init() {
 
 	utilruntime.Must(otelv1alpha1.AddToScheme(scheme))
 	utilruntime.Must(routev1.AddToScheme(scheme))
+	utilruntime.Must(monitoringv1.AddToScheme(scheme))
```

Comment:
Should we also put this behind the flag?

Reply:
When this code is executed, the flag value is unknown.